curriculum/challenges/english/blocks/workshop-colorful-boxes/68ef0d6d156f86a6ac1824c6.md

```diff
@@ -0,0 +1,86 @@
+---
+id: 68ef0d6d156f86a6ac1824c6
+title: Step 11
+challengeType: 0
+dashedName: step-11
+---
+
+# --description--
+
```

Comment:
I think this is a bit misleading. `align-content` controls the arrangement of items along the cross axis. This is something that should be mentioned explicitly. Also, at this stage it seems `justify-content` is the right property to set in order to actually see something on the page.